docs/iris/src/installing.rst

```diff
@@ -41,11 +41,45 @@ need the Iris sample data. This can also be installed using conda::
 Further documentation on using conda and the features it provides can be found
 at https://conda.io/en/latest/index.html.
 
+.. _installing_from_source_without_conda:
+
+Installing from source without conda on Debian-based Linux distros (devs)
+-------------------------------------------------------------------------
+
+Iris can also be installed without a conda environment. The instructions in
+this section are valid for Debian-based Linux distributions (Debian, Ubuntu,
+Kubuntu, etc.).
+
+Iris and its dependencies need some shared libraries in order to work properly.
+These can be installed
+with apt::
+
+  sudo apt-get install python3-pip python3-tk libudunits2-dev libproj-dev proj-bin libgeos-dev libcunit1-dev
+  
+Consider executing::
+
+  sudo apt-get update
+  
+before and after installation of Debian packages.
+
+The rest can be done with pip. Begin with numpy::
+
+  pip3 install numpy
+
+Finally, Iris and its Python dependencies can be installed with the following
+command::
+
+  pip3 install astropy tqdm oktopus bs4 setuptools nc_time_axis cython cfunits cf-units cf-python pandas stratify pyugrid scitools-pyke black imagehash nose asv scitools-iris
```

Comment:
@MHBalsmeier You seem to have quite a few extra dependencies and optional dependencies mixed in here... would it be possible to strip this example back to it's essential bare minimum?

I'm guessing that this is the package dependency requirements for your specific area of research/work.

The following appear to extra dependencies for your particular needs:
* `astropy`
* `tqdm`
* `oktopus`
* `bs4`
* `cython`
* `cf-python`
* `asv`

The following appear to be optional dependencies of `iris`, but not strictly necessary:
* `nc_time_axis`
* `pandas`
* `stratify`
* `pyugrid`
* `black`
* `imagehash`
* `nose`

I'm also assuming that `cfunits` isn't a necessary dependency for `cf-units`? (`cfunits` is a similar package as our SciTools `cf-units`, which is maintained by David Hassell from Reading University)

So, I guess my question is, could this actually boil down to simply:
```shell
> pip3 install setuptools cf-units scitools-pyke scitools-iris
```
Although, I'm guessing (off the top of my head) that `cython` is necessary for `cf-units`? If that's the case then we have:
```shell
> pip3 install setuptools cython cf-units scitools-pyke scitools-iris
```
Is it possible for you to confirm this?